cypher/cypher-docs/src/docs/dev/ql/administration/security/grant-deny-syntax.asciidoc

```diff
@@ -1,72 +0,0 @@
-The administrators can use Cypher commands to manage Neo4j graph administrative rights.
```

Comment:
Content was moved to neo4j-manual-modeling/4.3/cypherManual/asciidoc/access-control/manage-privileges.adoc

https://neo4j.com/docs/cypher-manual/4.3/access-control/manage-privileges/#access-control-graph-privileges